packages/api-client-core/src/operationBuilders.ts

```diff
@@ -195,7 +224,7 @@ export const backgroundActionResultOperation = <Action extends AnyActionFunction
       const selection = options?.select || action.defaultSelection;
 
       fields = {
-        [`... on ${resultType}`]: actionResultFieldSelection(action.modelApiIdentifier, selection, action.hasReturnType),
+        [`... on ${resultType}`]: actionResultFieldSelection(action.modelApiIdentifier, selection),
```

Comment:
Just to double check: this shouldn't be:

```ts
actionResultFieldSelection(action.modelApiIdentifier, selection, undefined, action.hasReturnType)
```

?

Reply:
yah good catch, updated to use the new call signature will all the metadata about the action